server/src/main/resources/org/cloudfoundry/identity/uaa/db/postgresql/V4_101_1639764160__Add_LowerIndex_To_Users_Wo_Origin.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX concurrently IF NOT EXISTS users_key_lower_wo_origin ON users (LOWER(username),LOWER(identity_zone_id));
```

Comment:
only one remark, all postgresql relavat commds are in upper case exception concurrently. so you can think about this but no functional change